Simplenote/Classes/NoteOptionsViewController.swift

```diff
@@ -155,13 +215,14 @@ class NoteOptionsViewController: UITableViewController {
     fileprivate var collaborationSection: Section {
         let rows = [
             Row(style: .Value1,
-                configuration: { (cell: UITableViewCell, row: Row) in
+                configuration: { [note, collaboratorNumberFormatter] (cell: UITableViewCell, row: Row) in
                     let cell = cell as! Value1TableViewCell
                     cell.textLabel?.text = NSLocalizedString("Collaborate", comment: "Note Options: Collaborate")
-                    cell.accessibilityLabel = NSLocalizedString("Tap to open collaboration menu", comment: "Accessibility hint on cell that opens collaboration menu")
+                    cell.detailTextLabel?.text = collaboratorNumberFormatter.string(from: NSNumber(value: note.emailTagsArray.count))
```

Comment:
Note here: When there are no collaborators, this would effectively display a `0` on the right hand side.

Do you think we could either hide the number if it's zero, or display a `-` instead perhaps?